static/js/util/sorting.js

```diff
@@ -19,9 +20,13 @@ export function resumeOrder(entries: SortableEntry[], dateFieldName: string): So
   return entries.sort(sortFunc);
 }
 
-export function sortWorkEntriesByDate(entries: Array<WorkHistoryEntry>): Array<WorkHistoryEntry> {
+export function workEntriesByDate(entries: Array<WorkHistoryEntry>): Array<WorkHistoryEntry> {
   let sorted = [];
   sorted.push(...resumeOrder(entries.filter(entry => entry.end_date === null), 'start_date'));
   sorted.push(...resumeOrder(entries.filter(entry => entry.end_date !== null), 'end_date'));
   return sorted;
 }
+
+export function educationEntriesByDate(entries: Array<EducationEntry>): Array<EducationEntry> {
+  return resumeOrder(_.cloneDeep(entries), 'graduation_date');
```

Comment:
It's generally bad practice to modify arguments passed into a function unless explicitly documented somehow. Can you change `resumeOrder` to do the list copy there and remove the `_.cloneDeep` here?